app/views/partials/_view_data.html.erb

```diff
@@ -36,8 +36,74 @@
           </div>
         </div>
       </div>
-    <div class="tab-pane fade show-page-views-tab-pane" id="profile" role="tabpanel" aria-labelledby="profile-tab">This exciting new feature is coming soon! </div>
+    <div class="tab-pane fade show-page-views-tab-pane" id="profile" role="tabpanel" aria-labelledby="profile-tab">
+      <!-- Don't touch this please, map projections are made for 960x500 aspect ratio -->
```

Comment:
I think the following code would give an array of hashes for each order (the same format as d3 parsing a tsv file gives you, I think?)
<div style="display: none;" id="auniquejsonid">
  [<% @query.orders.each_with_index do |order, index| %>
    <%= order.to_h %><%= if index < @query.orders.count ? ", " : "" %>
  <% end %>]
</div>